subsys/bluetooth/controller/ll_sw/nordic/lll/lll_scan_aux.c

```diff
@@ -1051,18 +1062,33 @@ static int isr_rx_pdu(struct lll_scan *lll, struct lll_scan_aux *lll_aux,
 	/* Passive scanner or scan responses */
 #if defined(CONFIG_BT_CENTRAL)
 	} else if (!lll->conn &&
-		   lll_scan_ext_tgta_check(lll, false, false, pdu, rl_idx)) {
+		   ((lll_aux && lll_aux->is_chain_sched) ||
+		    (lll->lll_aux && lll->lll_aux->is_chain_sched) ||
```

Comment:
Under what conditions is it possible to get here when only one of these conditions is not true `lll_aux && lll_aux->is_chain_sched`  or `lll->lll_aux && lll->lll_aux->is_chain_sched`?

Reply:
One more question here. `lll_aux == NULL` and lll->lll_aux == NULL` may happen only in case of execution path started by LLL primary scan context and a function `ull_scan_aux_setup()` was not called by ULL in the meantime (between LLL primary scan context and call to isr_rx_pdu) or the function was not able to allocate lll_scan_aux . It that correct?